desktop/nodeManager.js

```diff
@@ -3,13 +3,26 @@ import os from 'os';
 import { ipcConsts } from '../app/vars';
 
 const child = require('child_process').execFile;
+const find = require('find-process');
 
 const osTargetNames = {
   Darwin: 'mac',
   Linux: 'linux',
   Windows_NT: 'windows'
 };
 
+const getPidByName = ({ name }) => {
+  return new Promise((resolve, reject) => {
+    try {
```

Comment:
there are 2 options to write this function:
`find .... then(), (err) { reject(); }`
or
`try {
   const list = await find
   ...
}
 catch {}
`
but not mixing two ways to deal async response